libraries/google_container_regional_cluster.rb

```diff
@@ -92,10 +92,12 @@ def parse
     @logging_service = @fetched['loggingService']
     @monitoring_service = @fetched['monitoringService']
     @network = @fetched['network']
+    @network_self_link = @fetched['networkConfig']['network']
```

Comment:
Thanks, this looks like a great addition.  We have two types of resources in InSpec GCP - handwritten and Magic Modules.  This resource happens to be generated via Magic Modules so unless this change is made there, it would be overwritten in future updates.  There's relevant discussion in this ticket - https://github.com/inspec/inspec-gcp/issues/162 but feel free to reach out to @slevenick or myself with any questions and we'll be happy to help!

Reply:
@skpaterson - I will take some time to understand how to use Magic Module to make this change.

Is this something related to what I am trying to achieve  - https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/resources/resource_container_cluster.go.erb#L1287?

For now, I will remove this change from this PR.